junit-runtime/src/main/scala/com/novocode/junit/Ansi.scala

```diff
@@ -0,0 +1,53 @@
+package com.novocode.junit
+
+object Ansi {
+  // Standard ANSI sequences
+  private[this] val NORMAL = "\u001B[0m"
+  private[this] val HIGH_INTENSITY = "\u001B[1m"
+  private[this] val LOW_INTESITY = "\u001B[2m"
+  private[this] val BLACK = "\u001B[30m"
+  private[this] val RED = "\u001B[31m"
+  private[this] val GREEN = "\u001B[32m"
+  private[this] val YELLOW = "\u001B[33m"
+  private[this] val BLUE = "\u001B[34m"
+  private[this] val MAGENTA = "\u001B[35m"
+  private[this] val CYAN = "\u001B[36m"
+  private[this] val WHITE = "\u001B[37m"
```

Comment:
Don't you think this is overkill, if you don't even use all of them and they are private? I think inlining the constants with a comment is just fine. (Maybe have one for `NORMAL`).